### DISCUSSION TITLE
What next ?

### DISCUSSION THREAD

**Discussion by domokane** [domokane/FinancePy#129] [General]
I am considering what to do next with this library. 

I use it for my teaching and I also have a number of students using it for various projects. 

But what I would like to know is 

1) Who is using it ?
2) Why you are using it ?
3) What are you doing with it ?
4) What could be improved ?

Any feedback would be very helpful.

**Comment by piterbarg** [comment]
Hi Dominic, Vladimir here. I have signed up for teaching a Math Finance Masters course "Topics in Derivatives Pricing". I thought I would use your library to demonstrate various points, at the moment focusing on interest rate curve building and usage. The library looks great btw! Hope you are well.

**Reply by domokane** [reply]
Hi Vladimir
Nice to hear from you ! I am very well. Hope you are too.
Happy for you to use the library and I hope you can find it useful. 
Best
Dominic
